nbs/06_transform.ipynb

```diff
@@ -560,9 +560,9 @@
     "        For each clone of an LHS node (apart from the first one), add it to the input graph \n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #37.        lhs_strs = lhs.split('^')
</pre><p>again, this should be in the parser</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/graph_rewrite/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>On it, thanks!</p>